### PR TITLE
feat: add hall page

### LINKED ISSUE #168: feat(guide): create Hall Info page
The design is available on [figma](https://www.figma.com/file/VtzXViXjXah3IuWn2xBRae/Website?node-id=736%3A51).

Hall Info

### PR DESCRIPTION
**Pull request by Amal-Mohan-2002**
### Your checklist for this pull request

🚨Please review the [guidelines for contributing](https://github.com/Monday-Morning/project-tahiti/blob/main/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start _your branch_ off _default branch_.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Linked Issues

Fixes #168 

### Description
image
image
image
image
image

Please describe your pull request.

❤️Thank you!

### Post merge checklist

- [x] Follow steps from the [guidelines for contributing](https://github.com/Monday-Morning/project-tahiti/blob/main/CONTRIBUTING.md) to this repository.
- [x] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

File: client/src/components/article/ArticleTable.js [hunk 1]
```diff
@@ -67,4 +67,5 @@ const useStyles = makeStyles((theme) => ({
   lighter: {
     fontWeight: 'regular',
   },
+
```
Comment (120EE0692) [inline client/src/components/article/ArticleTable.js:70]: @Amal-Mohan-2002 why changes here please undo this.

File: client/src/components/hallInfo/Info.js [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+                            className={classes.normalText}
+                          >
+                            <CallIcon className={classes.phoneIcon} />
+                            {item.phone}
+                          </Typography>
+                        )}
+                      </div>
+                    ))}
+                  </div>
+                </div> */}
```
Comment (120EE0692) [inline client/src/components/hallInfo/Info.js:101]: @Amal-Mohan-2002 remove these comment if of no use.

File: client/src/components/hallInfo/Info.js [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+  },
+  hnc: {
+    display: 'flex',
+    gap: '10px',
+    justifyContent: 'space-between',
+    margin: '15px',
+    [theme.breakpoints.down('sm')]: {
+      flexDirection: 'column',
+    },
+  },
```
Comment (120EE0692) [inline client/src/components/hallInfo/Info.js:194]: @Amal-Mohan-2002 remove this if of no use.

File: client/src/components/hallInfo/Info.js [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+    padding: '5px',
+  },
+  info: {
+    padding: '10px',
+  },
+  hncTitle: {
+    fontWeight: '700',
+    fontSize: '17px',
+    lineHeight: '24px',
+  },
```
Comment (120EE0692) [inline client/src/components/hallInfo/Info.js:225]: @Amal-Mohan-2002 remove this if of no use.

**Review by 120EE0692 — CHANGES_REQUESTED** [review]
@Amal-Mohan-2002 change title of the PR to feat: add hall page

File: client/src/components/hallInfo/Info.js (hunk 2 above)
Comment (Amal-Mohan-2002) [reply client/src/components/hallInfo/Info.js:101]: This part renders the HNC section.
Should I remove this?

File: client/src/components/article/ArticleTable.js [hunk 5]
```diff
@@ -67,4 +67,4 @@ const useStyles = makeStyles((theme) => ({
   lighter: {
     fontWeight: 'regular',
   },
-}));
+}));
```
Comment (120EE0692) [inline client/src/components/article/ArticleTable.js:70]: @Amal-Mohan-2002 undo this change.

**Review by 120EE0692 — CHANGES_REQUESTED** [review]

File: client/src/components/widgets/DropDownCard.js [hunk 6]
```diff
@@ -56,5 +56,8 @@ const useStyles = makeStyles((theme) => ({
     lineHeight: '32px',
     color: '#222222',
     paddingTop: '6px',
+    [theme.breakpoints.down('sm')]: {
+      fontSize: '20px',
+    },
   },
-}));
+}));
```
Comment (120EE0692) [inline client/src/components/widgets/DropDownCard.js:63]: @Amal-Mohan-2002 run prettier in this file

File: client/src/pages/hallInfo.jsx [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+        />
+      </Head>
+      <Marginals>
+        <HallInfo />
+      </Marginals>
+    </>
+  );
+}
+
+export default HallInfos;
```
Comment (120EE0692) [inline client/src/pages/hallInfo.jsx:77]: @Amal-Mohan-2002 run prettier in this file.

File: client/src/screens/HallInfo.js [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+
+export default HallInfo;
+
+const useStyles = makeStyles((theme) => ({
+  head: {
+    color: theme.palette.primary.blue60,
+    lineHeight: '2rem',
+    margin: '2.7rem 0 2rem 0.7rem',
+  },
+}));
```
Comment (120EE0692) [inline client/src/screens/HallInfo.js:35]: @Amal-Mohan-2002 run prettier in this file as well.

**Review by 120EE0692 — CHANGES_REQUESTED** [review]

**Comment by sambit-sankalp** [comment]
@Amal-Mohan-2002, update on this PR?

File: client/src/assets/placeholder/hallInfo.js [hunk 9]
```diff
@@ -0,0 +1,357 @@
...
+          established: '2008',
+          rooms: '1014',
+          boarders: '1200',
+          type: 'Boys Hostel with single and double-seater rooms',
+          pic: cvr,
+        },
+      ],
+    },
+    {
+      title: 'Mokshagundam Visvesvaraya (MV) Hall of Residence',
```
Comment (120EE0692) [inline client/src/assets/placeholder/hallInfo.js:319]: @Amal-Mohan-2002 remove (MV) only keep 
Mokshagundam Visvesvaraya Hall of Residence.

**Comment by 120EE0692** [comment]
@Amal-Mohan-2002 make to write your commit message in small caps.

**Review by 120EE0692 — APPROVED** [review]

**Review by sambit-sankalp — APPROVED** [review]
GTG

**Review by rutajdash — APPROVED** [review]

**Comment by rutajdash** [comment]
@Amal-Mohan-2002 please update your branch with latest code

**Review by 120EE0692 — APPROVED** [review]

**Pull request merged**